scipy/sparse/linalg/_onenormest.py

```diff
@@ -450,17 +450,17 @@ def _onenormest_core(A, AT, t, itmax):
         if t > 1:
             # (5)
             # Break if the most promising t vectors have been visited already.
-            if np.in1d(ind[:t], ind_hist).all():
+            if np.isin(ind[:t], ind_hist).all():
```

Comment:
Looking through this code, `ind` is 1-dimentional so `np.in1d` should be replaceable by `np.isin` without `.ravel()`.